layouts/shortcodes/config-uri-k8s-istio.html

```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/kubeflow/kubeflow/v0.6-branch/bootstrap/config/kfctl_k8s_istio.0.6.2.yaml
```

Comment:
@jlewi Is this the right file for the `kfctl-k8s-istio` config? You can see the result in the preview here:
https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-k8s-istio/